docs/contributing_dev.md

```diff
@@ -0,0 +1,191 @@
+
+## Contributing
+
+We expect contributors to abide by our underlying
+[code of conduct](https://dev.to/code-of-conduct). All conversations and
+discussions on GitHub (issues, pull requests) and across
+[dev.to](https://dev.to) must be respectful and harassment-free.
+
+Remember that communication is the lifeblood of any Open Source project. We are
+all working on this together, and we are all benefiting from this software. It's
+very easy to misunderstand one another over asynchronous, text-based
+conversations: When in doubt, assume everyone you're interacting within this
+project has the best intentions.
+
+If you feel another member of the community has violated our Code of Conduct,
+you may anonymously contact the team with our
+[abuse report form](https://dev.to/report-abuse).
+
+### Where to contribute
+
+All [issues](https://github.com/thepracticaldev/dev.to/issues) labeled
+[ready for dev](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22)
+and
+[type: bug](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22)
+all are up for grabs.
+
+- [good first issue](https://github.com/thepracticaldev/dev.to/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+)
+  are issues meant for newer developers.
+- [difficulty: easy](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+easy%22)
+  issues are usually confined to isolated areas of existing code.
+- [difficulty: medium](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+medium%22)
+  issues are usually can sometimes entail new features and might affect a
+  significant area of the codebase, but aren't overly complex.
+- [difficulty: hard](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+hard%22)
+  issues will typically be far-reaching, and might require architecture
+  decisions during implementation. This label might also denote highly complex
+  issues.
+
+While PRs without an associated `ready for dev` or `type: bug` labeled issue may
+still be merged, please note that the core team will prioritize PRs that solve
+existing issues. We strongly encourage creating an issue before working on a PR!
+
+When in doubt, ask a
+[core team member](https://github.com/thepracticaldev/dev.to/#core-team) by
+mentioning us on the issue.
+
+**Documentation** is almost always a great place to start contributing to a new
+project. DEV is an Open Source, community-driven project, so providing and
+maintaining quality documentation is one of our most important jobs. You can
+find more information about contributing to the documentation in the
+[docs/ directory](https://github.com/thepracticaldev/dev.to/blob/master/docs/contributing.md)!
+
+**Refactoring** code, or improving the code without modifying the behaviour, is
```

Comment:
Since I noticed "behaviour" here: I was under the impression that we generally use American English spellings? I think we should stay consistent in this regard.